backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -72,7 +73,8 @@ public FEList load(Concept<?> concept) throws Exception {
 
 	private final LoadingCache<Pair<AbstractSelectFilter<?>, String>, List<FEValue>> searchCache =
 			CacheBuilder.newBuilder()
-						.expireAfterAccess(Duration.ofMinutes(2))
+						.softValues()
+						.weakKeys()

```

Comment:
Bei dem nodeCache zieht "identity comparision" denke ich gut, aber hier bei diesem cache werden die Keys on the fly erzeugt.

Reply:
was genau meinst du?

Reply:
Das der Cache eingehende Keys nicht mit `.equals` sondern mit `==` vergleicht. Das ist problematisch, weil in `#autocompleteTextFilter` immer ein neues Objekt erzeugt wird bei einem Lookup:
```
fullResult = searchCache.get(Pair.of(filter, text));
```

Reply:
ah, gut gesehen, ich hab dann mal die weakKeys entfernt, dann sollte es doch passen oder nicht?

Reply:
Jap, softValues sollte okay sein, denn wir rufen ja kein contains oder delete mit einem selbst konstruierten Value auf